builder/steps/gen-dockerfile/contents/src/data/install-node-version

```diff
@@ -0,0 +1,4 @@
+# Check to see if the the version included in the base runtime satisfies
+# %s, if not then do an npm install of the latest available
```

Comment:
'npm install of the latest version of node' doesn't make any sense. Is `nvm` implied here? But we don't use nvm either.